modules/mailjet_simple_node_campaigns/mailjet_simple_node_campaigns.module

```diff
@@ -139,7 +139,23 @@ function mailjet_simple_node_campaigns_form_node_form_submit($form, &$form_state
       $arguments['SenderEmail'] = check_plain($form_state['values']['node_campaign_settings_overrides']['node_campaign_email']);
     }
 
-    mailjet_simple_node_campaigns_prepare_campaign_draft($list_id, $title, $message, $template_id, $arguments);
+    // Specify the text format to be used when sanitizing the message.
+    $advanced_settings = mailjet_simple_node_campaigns_get_setting('advanced_settings');
+    $text_format = $advanced_settings['text_format'] ? $advanced_settings['text_format'] : NULL;
+
+    // Send the node campaign.
+    mailjet_simple_node_campaigns_prepare_campaign_draft($list_id, $title, $message, $template_id, $text_format, $arguments);
+  }
+}
+
+/**
+ * Helper function that returns the specified node campaign setting.
+ */
+function mailjet_simple_node_campaigns_get_setting($setting) {
+  if (!empty($setting)) {
+    $settings = variable_get('mailjet_simple_node_campaigns_settings');
+    $result = $settings[$setting];
```

Comment:
need to check empty here. 

return !empty($settings[$setting]) ? $settings[$setting] : FALSE

Reply:
Might suggest making this a dual purpose method


function ____get_settings($setting = NULL)...

if (!empty($setting).... get specific settings
otherwise return all settings.